template/macros.jinja

```diff
@@ -43,22 +42,30 @@
             - salt://xxx/files/default/etc/yyy/zzz.conf.jinja
           - template: jinja
   #}
-  {%- set path_prefix = prefix|replace(':', '/') %}
-  {%- set files_switch_list = salt['pillar.get'](prefix ~ ':files_switch',
-                                           default_files_switch) %}
+  {%- set formula = 'template' %}
+  {%- set path_prefix = salt['config.get'](formula ~ ':tofs:path_prefix', formula) %}
```

Comment:
Using `config.get` to make the `path_prefix` configurable or use `formula` name as the default.